assets/src/components/propertiesPanel/miniMap.tsx

```diff
@@ -0,0 +1,68 @@
+import { DomEvent } from "leaflet"
+import React, { useContext, useEffect } from "react"
+import { Link } from "react-router-dom"
+import { StateDispatchContext } from "../../contexts/stateDispatchContext"
+import { useStations } from "../../hooks/useStations"
+import { Vehicle, VehicleId } from "../../realtime"
+import { Shape, Stop } from "../../schedule"
+import { setSelectedVehicle } from "../../state/searchPageState"
+import inTestGroup, { MAP_BETA_GROUP_NAME } from "../../userInTestGroup"
+import { mapModeForUser } from "../../util/mapMode"
+import Map from "../map"
+
+const SearchMapLink = ({ vehicleId }: { vehicleId: VehicleId }) => {
+  const [_state, dispatch] = useContext(StateDispatchContext)
+  const ref = React.useRef(null)
+
+  useEffect(() => {
+    DomEvent.disableClickPropagation(ref.current!)
+  }, [])
```

Comment:
**question:** thinking through React component lifecycle considerations here - is there any situation in which `ref.current` would change and we would want to fire the effect again?

Reply:
Thanks for asking, I hadn't thought about where this fit into the lifecycle. 

I wouldn't expect `ref.current` to change, but maybe that is an assumption about the inner-workings of `Link`, which may not be a safe assumption. 

I can opt to follow the guidance of this [react doc](https://reactjs.org/docs/hooks-faq.html#how-can-i-measure-a-dom-node) and [useCallback may be what you meant by useRef & useEffect](https://medium.com/welldone-software/usecallback-might-be-what-you-meant-by-useref-useeffect-773bc0278ae) and switch to useCallback instead.

Reply:
Ah okay, I wasn't actually away of callback refs - after looking at those links, though, I like the approach.